canonical-web-and-design/jaas-dashboard
# Switch Header component to Typescript

## Done

Switch Header component to Typescript

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check code
- Make sure all tests pass
